icu4c/source/common/unicode/stringpiece.h

```diff
@@ -77,6 +80,29 @@ class U_COMMON_API StringPiece : public UMemory {
    */
   StringPiece(const std::string& str)
     : ptr_(str.data()), length_(static_cast<int32_t>(str.size())) { }
+#ifndef U_HIDE_DRAFT_API
+  /**
+   * Constructs from some other implementation of a string piece class, from any
+   * C++ record type that has these two methods:
+   *
+   *   struct OtherStringPieceClass {
```

Comment:
Is this indentation sufficient for doxygen to turn it into a `<pre>` block?

Reply:
No, apparently it was not. I ran Doxygen and it seems as if with the configuration used for ICU4C there isn't any indentation sufficient to automatically turn this into a code block. I've updated to use a `\code` annotation now instead.

Reply:
Shane has successfully used a relative indent of 4 spaces to make it work. But `\code` is good, too.